logic/src/commonMain/kotlin/com/wire/kalium/logic/data/client/ClientRepository.kt

```diff
@@ -253,6 +253,7 @@ class ClientDataSource(
         newClientEvent: Event.User.NewClient
     ): Either<CoreFailure, Unit> = wrapStorageRequest {
         newClientDAO.insertNewClient(clientMapper.toInsertClientParam(selfUserID, newClientEvent))
+        clientDAO.insertClient(clientMapper.toInsertClientParam(selfUserID, newClientEvent))
     }
```

Comment:
can we do this on the DB level

insertNewSelfClient {

-- insert into new client table 

-- insert into clients table

}

Reply:
No need for triggers in this case, it will overcomplicate it. 
a simple transaction should do the trick.